### ISSUE TITLE
Firmware

### ISSUE THREAD

**Issue by Kaevandaja** [pembem22/jbl-firmware#6]
Can i download 2.3.0 csr firmware to Qcc? I Really want lfm back, or is there something i can to for get back?

**Comment by pembem22** [comment]
No, you can only flash CSR firmware to CSR speakers. I don't if there exists a way to flash QCC firmware.
